xml/System.Xml/XmlException.xml

```diff
@@ -347,9 +347,39 @@ The `XmlException` is a generic XML-related error that is thrown if problems occ
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the line number indicating where the error occurred.</summary>
-        <value>The line number indicating where the error occurred.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the 1-based line number indicating where the error occurred.</summary>
```

Comment:
I think something like this would be easier to read.

```suggestion
        <summary>Gets the line number indicating where the error occurred. Line numbers start at 1.</summary>
```